packager/accept/accept.go

```diff
@@ -18,23 +18,57 @@ const SxgContentType = "application/signed-exchange;v=" + AcceptedSxgVersion
 // signedexchange library.
 var SxgVersion = version.Version1b3
 
+// Tokenize a comma-separated string of accept patterns into a slice
+func tokenize(accept string) []string {
+	var tokens []string
+	acceptLen := len(accept)
+	if acceptLen == 0 {
+		return tokens
+	}
+
+	inQuotes := false
+	startIndex := 0
+	for i := 0; i < acceptLen; i++ {
+		char := accept[i]
+		switch char {
+		case '"':
+			inQuotes = !inQuotes;
+		case ',':
+			if !inQuotes {
+				tokens = append(tokens, strings.TrimSpace(accept[startIndex:i]))
+				startIndex = i + 1;
+			}
+		case '\\':
+			i++
```

Comment:
I think if `!inQuotes`, this should be a parse error, per the [tchar definition](https://tools.ietf.org/html/rfc7230#section-3.2.6). I guess returning empty list would make sense in this case. Add a test for this if possible; maybe something like `application/signed-exchange;v=b3;x=foo\,bar`.

(Perhaps `mime.ParseMediaType` already catches this? Ditto above; I'd prefer precision / defense in depth.)

Reply:
This is addressed in b974b06fab6008e2e0c173060108eed6b4e2ec7c